### PR TITLE
[release-1.3] fix: update janus-idp/cli in wrappers to 1.13.3

### PR DESCRIPTION
**Pull request by kadel**
## Description

Update janus-idp/cli in wrappers to the 1.13.3 that include fix for  https://issues.redhat.com/browse/RHIDP-3574

## Which issue(s) does this PR fix

- Fixes https://issues.redhat.com/browse/RHIDP-3574

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins/wrappers/backstage-community-plugin-3scale-backend/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
   },
   "devDependencies": {
     "@backstage/cli": "0.26.11",
-    "@janus-idp/cli": "1.13.2"
+    "@janus-idp/cli": "1.13.3"
```
[the remark this answers is not in the document]
Comment (kadel) [reply dynamic-plugins/wrappers/backstage-community-plugin-3scale-backend/package.json:43]: That cleans the directory with the derived package, right? This is not saved anywhere in the git. As far as I know, it is needed here.

**Review by nickboldt — APPROVED** [review]

**Pull request merged**